File: Entregavel2/Java/Fibonacci.java
package Java;

import java.util.Scanner;

public class Fibonacci{
    int n, a = 0, b = 1, i = 2;
    Scanner prompt = new Scanner(System.in);
        public void sequenciaFibonacci() {
            System.out.println("Numero de termos: ");
            int n = prompt.nextInt();
            while(i<n)  {
                System.out.print(a + ", " +  b + ", ");
                a = a + b;
                b = a + b;
                i = i + 2;
            }
            if (i == n)
                System.out.print(a + ", " +  b);
            else
                System.out.print(a);
            
        }   
}

File: Entregavel2/JavaScript/fibonacci.js
function calcularFibonacci(termos){
    var sequencia = [0, 1];
    for (var i = 2; i < termos; i++){
        var proximoTermo = sequencia[i - 1] + sequencia[i - 2];
        sequencia.push(proximoTermo);
    }
    return sequencia.slice(0, termos);
}

document.getElementById("calcularFibonacci").addEventListener("click", function() {
    const quantidadeTermos = parseInt(document.getElementById("Fibonacci").value);

    if(isNaN(quantidadeTermos) || quantidadeTermos < 0){
        document.getElementById("resultado").textContent = "Por favor, insira um numero valido";
        return;
    }

    const resultadoFibonacci = calcularFibonacci(quantidadeTermos);
    document.getElementById("resultado").textContent = resultadoFibonacci.join(", ");

})

document.getElementById("limpar").addEventListener("click", function() {
    document.getElementById("resultado").textContent = "";
})